Makefile

```diff
@@ -41,6 +42,13 @@ integration-test:
 		--junit-xml=$(TEST_RESULTS_DIR)/integration_test_results.xml \
 		-v $(PYTEST_ARGS)
 
+notification-test:
+	$(DOCKER_COMPOSE) -f $(NOTIFICATION_TEST_DIR)/docker-compose.yml build
+	$(DOCKER_COMPOSE) -f $(NOTIFICATION_TEST_DIR)/docker-compose.yml up -d server
+	ENV=$(ENV) $(DOCKER_COMPOSE) -f $(NOTIFICATION_TEST_DIR)/docker-compose.yml run -it --name notification-tests tests
+	docker cp notification-tests:/code/notification-tests.xml $(NOTIFICATION_TEST_DIR)
+	docker rm notification-tests
+
```

Comment:
The tests aren't passing for me locally... I get the feeling that part of the problem is we don't incorporate poetry in this command. Could we try something like:

```suggestion
notification-test: $(INSTALL_STAMP)
	$(DOCKER_COMPOSE) -f $(NOTIFICATION_TEST_DIR)/docker-compose.yml build
	$(DOCKER_COMPOSE) -f $(NOTIFICATION_TEST_DIR)/docker-compose.yml up -d server
	$(POETRY) run docker compose -f $(NOTIFICATION_TEST_DIR)/docker-compose.yml run -it --name notification-tests tests
	docker cp notification-tests:/code/notification-tests.xml $(NOTIFICATION_TEST_DIR)
	docker rm notification-tests

```

Errors:
<img width="1274" alt="image" src="https://github.com/user-attachments/assets/214dadc3-d925-4291-99c9-35a31b6ac2df">

Reply:
They have to be run inside of docker, see the `README`.

Reply:
Oh wait, I think those steps there are wrong, let me fix.

Reply:
Please try again, I think I fixed it

Reply:
Okay I've cleared the amd/arm issue see https://github.com/mozilla-services/autopush-rs/pull/789/commits/0e0372be76bc25c8418045f272fefcd46289f3ea

But still the tests don't pass, new errors:
<img width="1287" alt="image" src="https://github.com/user-attachments/assets/44fc267a-ebad-4a1a-8363-9f725af26514">